hadoop-hdfs-project/hadoop-hdfs-client/src/main/java/org/apache/hadoop/fs/XAttr.java

```diff
@@ -158,6 +158,6 @@ public boolean equalsIgnoreValue(Object obj) {
   @Override
   public String toString() {
     return "XAttr [ns=" + ns + ", name=" + name + ", value="
-        + Arrays.toString(value) + "]";
+        + new String(value, Charset.defaultCharset()) + "]";
```

Comment:
Not sure if all attributions could be readable. If not, it will print disorderly code? Do you test any attributions for this improvement? Thanks.

Reply:
agreed, xattr is not necessarily printable. For example, encryption zone file EDEK. To print, have we considered using XattrCodec APIs?

Reply:
Thanks @Hexiaoqiao & @jojochuang for quick review & feedback.
I faced this issue while printing custom properties on debugging some issue.
I tried to print the xAttr directly as it has toString implementation. We can make use of AXttrCodec utility for conversion.

Printing EDEKs or any other sensitive is vulnerable? If so, we can remove the bytes printing in toString. Otherwise, we shall close the issue directly? Thank you.